sdcm/nemesis.py

```diff
@@ -3518,10 +3518,7 @@ def decommission_node(self, node):
     def decommission_nodes(self, add_nodes_number, rack, is_seed: Optional[Union[bool, DefaultValue]] = DefaultValue,
                            dc_idx: Optional[int] = None):
         for _ in range(add_nodes_number):
-            if self._is_it_on_kubernetes():
-                self.set_target_node(rack=rack, is_seed=is_seed, allow_only_last_node_in_rack=True)
-            else:
-                self.set_target_node(is_seed=is_seed, dc_idx=dc_idx)
+            self.set_target_node(rack=rack, is_seed=is_seed, allow_only_last_node_in_rack=True)
```

Comment:
Why in non-K8S cases we don't allow non-last node in a rack to be disrupted anymore?

Reply:
I think I misunderstood that - reverting this part.